websocket.go

```diff
@@ -19,59 +19,59 @@ type Client struct {
 	Hub    *Hub
 }
 
-func (client *Client) Receive() {
-	client.Conn.SetReadLimit(1024)
-	client.Conn.SetReadDeadline(time.Now().Add(pongWait))
-	client.Conn.SetPongHandler(func(string) error {
-		client.Conn.SetReadDeadline(time.Now().Add(pongWait))
+func (c *Client) Receive() {
+	c.Conn.SetReadLimit(1024)
+	c.Conn.SetReadDeadline(time.Now().Add(pongWait))
+	c.Conn.SetPongHandler(func(string) error {
+		c.Conn.SetReadDeadline(time.Now().Add(pongWait))
```

Comment:
So while researching, I discovered that there are some words that have a consistent form in go, words like client, hub, parameter, boolean ... it is a typical go naming convention instead of writing client, we can just use c, and instead of hub, we can use h. there were other instances that I could have done this as well, but I didn't want to make many changes so that it is still understandable.